packages/bundle/src/adaptiveCards/Attachment/ReceiptCardContent.js

```diff
@@ -34,7 +34,7 @@ const ReceiptCardContent = ({ actionPerformedClassName, content, disabled }) =>
       builder.addTextBlock(title, { size: TextSize.Medium, weight: TextWeight.Bolder, wrap: richCardWrapTitle });
 
       if (facts) {
-        const [firstFactColumn, lastFactColumn] = builder.addColumnSet([75, 25]);
+        const [firstFactColumn, lastFactColumn] = builder.addColumnSet(['75', '25']);
```

Comment:
this looks weird to me. the elements need to be strings that look like numbers?

Reply:
Yeah you're right. Looked at the code again and there's a way to provide the number instead of parsing a string. I'll push the fix up shortly.